src/home_robot/home_robot/utils/image.py

```diff
@@ -143,6 +142,7 @@ def depth_to_xyz(depth, camera: Camera):
 
 def to_o3d_point_cloud(xyz, rgb=None, mask=None):
     """conversion to point cloud from image"""
+    import open3d as o3d
```

Comment:
this is another duplicate function - can be filed as an issue

Reply:
Created issue here: https://github.com/facebookresearch/home-robot/issues/65